src/components/editor/ResourceTemplate.jsx

```diff
@@ -46,15 +46,20 @@ ResourceTemplate.propTypes = {
   retrieveResourceTemplate: PropTypes.func,
   resourceTemplateId: PropTypes.string,
   resourceTemplate: PropTypes.object,
-  title: PropTypes.string,
+  rtLabel: PropTypes.string,
   error: PropTypes.string,
 }
 
-const mapStateToProps = state => ({
-  resourceTemplate: rootResource(state),
-  title: state.selectorReducer.editor.rootResourceTitle,
-  error: state.selectorReducer.editor.serverError,
-})
+const mapStateToProps = (state, ownProps) => {
+  const resourceTemplate = rootResource(state)
+  const rtLabel = state.selectorReducer.entities.resourceTemplates[ownProps.resourceTemplateId]?.resourceLabel
+  const error = state.selectorReducer.editor.serverError
+  return {
+    resourceTemplate,
+    rtLabel,
+    error,
+  }
+}
```

Comment:
for experimentation, using this alternate form for the method was easier.  I just kept it.  I don't know why "selected" appears sometimes in that return section, but it was messing things up here so I removed it.